### PR TITLE
Add Settings and Scroll for What I'm working on for See Full Signature addon (Now renamed)

### LINKED ISSUE #295: Add Settings and Scroll for What I'm working on for See Full Signature addon (Now renamed)
**Describe the new addon**

Add Settings and Scroll for What I'm working on for See Full Signature addon (Now renamed)

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
**Resolves**

Resolves #295 

**Changes**

This adds Options to the addon, and a scroll for What I'm working on on the profiles.
**Reason for changes**

**Tests**

Inspect.

### CODE REVIEW

File: addons/full-signature/addon.json [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
+    }
+  ],
   "userstyles": [
     {
       "url": "style.css",
-      "matches": ["https://scratch.mit.edu/discuss/topic/*"]
+      "matches": ["https://scratch.mit.edu/discuss/topic/*"],
+      "setting_match": {
+        "setting_id": "signature",
+        "setting_value": "true"
```
Comment (WorldLanguages) [inline addons/full-signature/addon.json:29]: Whoops, that shouldn't be a string, but a boolean.

File: addons/full-signature/addon.json [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
+      }
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style2.css",
+      "matches": ["https://scratch.mit.edu/users/*"],
+      "setting_match": {
+        "setting_id": "whatworkingon",
+        "setting_value": "true"
```
Comment (WorldLanguages) [inline addons/full-signature/addon.json:39]: Here as well

**Comment by WorldLanguages** [comment]
Keep in mind you don't need to create an issue before a PR, if you've created the addon you don't need to auto-suggest features.

**Comment by GrahamSH-LLK** [comment]
@WorldLanguages Should be good to go!

**Comment by WorldLanguages** [comment]
Is there any reason you also add a scroll bar in the x axis?

**Comment by GrahamSH-LLK** [comment]
That was a mistake.

**Comment by WorldLanguages** [comment]
Lool I accidentally linted this PR instead of another, that makes sense

**Pull request merged**